common/commands/src/validator/cosmwasm/generators/mixnet.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use clap::Parser;
+use log::{debug, info};
+
+use cosmwasm_std::Decimal;
+use mixnet_contract_common::{InitialRewardingParams, InstantiateMsg, Percent};
+use std::time::Duration;
+
+#[derive(Debug, Parser)]
+pub struct Args {
+    #[clap(long, empty_values = false)]
+    pub rewarding_validator_address: String,
+
+    #[clap(long, empty_values = false)]
+    pub vesting_contract_address: String,
+
+    #[clap(long, default_value = "unymt")]
+    pub rewarding_denom: String,
```

Comment:
Given that those 3 fields already have default values, as given by the env setup, I would say mark them as optional (`Option<String>`) without any `clap` default values. And in the `generate` function bellow, use the values from `Args` or, if not specified, the values from environment (e.g. `std::env::var(network_defaults::var_names::REWARDING_VALIDATOR_ADDRESS)`)